project/src/com/company/personservices/PersonService.java

```diff
@@ -0,0 +1,181 @@
+package com.company.personservices;
+
+import com.company.Service;
+import com.company.accounts.CurrentAccount;
+import com.company.persons.Customer;
+import com.company.persons.Employee;
+import com.company.persons.Person;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonService implements PersonServiceInterface{
```

Comment:
[Fix] cred ca ar trebui sa renunti la acest serviciu cu totul deoarece Person ar trebui sa fie abstract, si nu ar trebui sa iti permita sa creezi sau sa modifici persoane

Reply:
Am vorbit pe Teams despre acest lucru si a ramas ca pot pastra AccountService si PersonService din moment ce ma folosesc de ele in serviciile claselor derivate. Am transformat clasele Person si Account in clase abstracte in urma acestul Pull Request, insa am ales sa pastrez serviciile acestor clase ca baza pentru serviciile claselor derivate(de exemplu pentru citirea unui Employee sau Customer ma voi folosi de aceeasi citire din PersonService, etc).

Source:
"Ma refer ca in loc sa scriu aceeasi citire a bazei Person in EmployeeService si CustomerService nu as putea sa las cum e acum si sa apelez direct citirea din PersonService chiar daca Person nu va avea instante? -> ok"